app/helpers/interactive_run_helper.rb

```diff
@@ -34,6 +34,13 @@ def interactive_data_div(interactive,run)
     if (run)
       data['puturl'] = put_url(interactive,run)
       data['geturl'] = get_url(interactive,run)
+      if run.collaboration_run
+        collaborator_urls = []
+        run.collaboration_run.collaborators_runs(run.activity, run.user).each do |collaborator_run|
+          collaborator_urls.push(get_url(interactive,collaborator_run))
```

Comment:
I see what you mean know about using `get_url` since we just use it in one place and use Javascript to add the domain.  I'll update the commit.